packages/analytics-types/src/campaign.ts

```diff
@@ -33,7 +33,14 @@ export interface CampaignTrackerOptions extends AttributionBrowserOptions {
 }
 
 export interface CampaignTracker extends CampaignTrackerOptions {
+  isNewCampaign(currentCampaign: Campaign, previousCampaign: Campaign): boolean;
```

Comment:
For types, I think it would make sense to have each plugin export its own types. Also each top level client could reexport the types to prevent the addition package import by end users. If we follow that model we could export these options from the MA plugin, and re-export them from analytics-ma-browser as part of the Config.

Reply:
Removed the types